packages/core-js/es/regexp/sticky.js

```diff
@@ -0,0 +1,5 @@
+require('../../modules/es.regexp.sticky');
+
+module.exports = function (it) {
+  return it.sticky;
+};
```

Comment:
Also required `stable/regexp/sticky`, `features/regexp/sticky, the same for `RegExp#test` and adding them to `tests/commonjs`.